setup/management.sh

```diff
@@ -27,39 +41,16 @@ if [ ! -d $venv ]; then
 	virtualenv -ppython3 $venv
 fi
 
-# pip<6.1 + setuptools>=34 had a problem with packages that
-# try to update setuptools during installation, like cryptography.
-# See https://github.com/pypa/pip/issues/4253. The Ubuntu 14.04
-# package versions are pip 1.5.4 and setuptools 3.3. When we used to
-# instal cryptography system-wide under those versions, it updated
-# setuptools to version 34, which created the conflict, and
-# then pip gets permanently broken with errors like
-# "ImportError: No module named 'packaging'".
-#
-# Let's test for the error:
-if ! python3 -c "from pkg_resources import load_entry_point" 2&> /dev/null; then
-	# This system seems to be broken already.
-	echo "Fixing broken pip and setuptools..."
-	rm -rf /usr/local/lib/python3.4/dist-packages/{pkg_resources,setuptools}*
-	apt-get install --reinstall python3-setuptools python3-pip python3-pkg-resources
-fi
-#
-# The easiest work-around on systems that aren't already broken is
-# to upgrade pip (to >=9.0.1) and setuptools (to >=34.1) individually
-# before we install any package that tries to update setuptools.
+# Upgrade pip because the Ubuntu-packaged version is out of date.
 hide_output $venv/bin/pip install --upgrade pip
```

Comment:
I got a warning about pip being out of date, is this just updating in the virtualenv? (I'm not a virtual env expert, or novice even :smile:)

Reply:
Yep, it's just updating in the virtualenv.